client/src/routes/app/[country]/[map]/[year]/initialize/LoadRegions.ts

```diff
@@ -73,6 +78,20 @@ function lockRegion(regionID: string) {
 }
 
 function loadRegions(node: HTMLDivElement): void {
+	/*Load Candidates from SVG, Candidates are defined in using the "candidates" property of the SVG document:
+	See usa.svg for a map that implements this. Please note that you must use &quot; instead of ". 
+	Edit the property in Inkscape to use double quotes. Inkspace will compress this to one line.
+	<svg
+	...
+   	candidates="[ 
+   		{&quot;id&quot;:&quot;0&quot;, &quot;name&quot;:&quot;Joe Custo Biden&quot;, &quot;margins&quot;: [{ &quot;color&quot;: &quot;#000055&quot; }, { &quot;color&quot;: &quot;#000099&quot; }, { &quot;color&quot;: &quot;#0000ff&quot; }]},
+   		{&quot;id&quot;:&quot;1&quot;, &quot;name&quot;:&quot;Donald Trump&quot;, &quot;margins&quot;: [{ &quot;color&quot;: &quot;#550000&quot; }, { &quot;color&quot;: &quot;#990000&quot; }, { &quot;color&quot;: &quot;#ff0000&quot; }]} 
+   	]"
+	*/
+	const candidatesStringified = node.querySelector('svg')?.getAttribute('candidates'); //This doesn't return SVG other than the map SVG
+	const candidates = candidatesStringified != null ? JSON.parse(candidatesStringified) : null; //If candidate property not set, set candidates to null so the next check knows to use default candidates.
+	if (candidates !== null) CandidatesStore.set(CandidateStoreSchema.parse(candidates)); //If no candidates are defined in SVG, use generics defined in stores/Candidates.ts
```

Comment:
1) this crashes if there is invalid JSON because JSON.parse with throw an error
2) this will also crash if your CandidateStoreSchema isn't matched

I was able to just throw everything in a try catch, and it will let the function finish executing if the data is incorrect.

![image](https://user-images.githubusercontent.com/13600696/209400972-08391bb8-db35-45cb-87bd-d5deed5e7172.png)

Reply:
The black color error happened when I was using 3 letter hex format. Like #FFF. Maybe we can look into fixing that in a later PR because it doesn't have to do with this feature.

Reply:
Wouldn't we want that "crashing" behavior to tell the mapmaker that their data format is incorrect? I suppose console.error gives the same debug info, but I left it this way since the only time invalid input would be given is when someone is making a map.

Reply:
I think displaying an error in the console, or a popup saying something went wrong would provide the user with a better experience.

Without catching the errors, it made the map some weird red color.

Reply:
Alright, lets go with the try catch then.